src/webapi/Controllers/EvaluationController.cs

```diff
@@ -98,52 +94,39 @@ public async Task<ActionResult<List<AvailableEvaluationObjective>>> GetEvaluatio
+            objectivesApi.Configuration.DefaultHeaders.Add("Content-Type", "application/json");
+            var tpdmEvaluationObjectives = await objectivesApi.GetEvaluationObjectivesAsync(limit: 100, offset: 0);
+            await _evaluationRepository.UpdateEvaluationObjectives(tpdmEvaluationObjectives.Select(teo => (EvaluationObjective)teo).ToList());
 
-                var peApi = new PerformanceEvaluationsApi(authenticatedConfiguration);
-                peApi.Configuration.DefaultHeaders.Add("Content-Type", "application/json");
-                var tpdmPerformanceEvaluations = await peApi.GetPerformanceEvaluationsAsync(limit: 100, offset: 0);
-                await _evaluationRepository.UpdatePerformanceEvaluations(tpdmPerformanceEvaluations.Select(pe => (PerformanceEvaluation)pe).ToList());
+            // Get Evaluation Elements which contain the EvaluationObjectiveTitles and update repository
+            var elementsApi = new EvaluationElementsApi(authenticatedConfiguration);
+            elementsApi.Configuration.DefaultHeaders.Add("Content-Type", "application/json");
+            var tpdmEvaluationElements = await elementsApi.GetEvaluationElementsAsync(limit: 100, offset: 0);
+            await _evaluationRepository.UpdateEvaluationElements(tpdmEvaluationElements.Select(tee => (EvaluationElement)tee).ToList());
 
-                // set next expiration time
-                var cachedValue = _memoryCache.GetOrCreate(
-                    dataExpirationKey,
-                    cacheEntry =>
-                    {
-                        cacheEntry.AbsoluteExpirationRelativeToNow = dataExpirationInterval;
-                        return DateTime.Now;
-                    });
-            }
-            return Ok();
-        }
+            var peApi = new PerformanceEvaluationsApi(authenticatedConfiguration);
+            peApi.Configuration.DefaultHeaders.Add("Content-Type", "application/json");
+            var tpdmPerformanceEvaluations = await peApi.GetPerformanceEvaluationsAsync(limit: 100, offset: 0);
+            await _evaluationRepository.UpdatePerformanceEvaluations(tpdmPerformanceEvaluations.Select(pe => (PerformanceEvaluation)pe).ToList());
 
-        // temporary for debugging
-        // TODO: Remove this catch block and add logging middleware EPPETA-25
-        catch (Exception ex)
-        {
-#pragma warning disable CA2200 // Rethrow to preserve stack details
-            return Problem(ex.Message);
-#pragma warning restore CA2200 // Rethrow to preserve stack details
+            // set next expiration time
+            var cachedValue = _memoryCache.GetOrCreate(
+                dataExpirationKey,
+                cacheEntry =>
+                {
+                    cacheEntry.AbsoluteExpirationRelativeToNow = dataExpirationInterval;
+                    return DateTime.Now;
+                });
         }
+        return NoContent();
```

Comment:
Here's a real change. Previously it was returning `Ok()` with no message. Better to return `NoContent` in that situation.